src/ChatWindow.vue

```diff
@@ -105,6 +137,10 @@ export default {
       type: String,
       default: ''
     },
+    chatListImageUrl: {
+      type: String,
+      default: ''
```

Comment:
It would be great if we could implement a fallback to some kind of initials in case there is no image url. I propose to keep this required for the moment. What do you think?
Maybe, when we will have such mechanism we will be able to make it optional and fall back to using initials.

Reply:
I think the best use case for chatListImageUrl is logged-in user icon. Also, the chats list looks great without an icon, so, I'd like to allow developers to have no icon for list of chats. In fact, I'm thinking about doing it myself. Defaulting to initials may be a great addition for icons in other places, but let's keep it out of scope of this pull request. In the future we could add a function that generates SVG with initials (do we have initials though?) and put it into the demo or put it inside the chat control and have some other way of specifying that icon with initials should be displayed (perhaps a special case value like 'initials' for a URL property?).